src/Common/src/Interop/Windows/Interop.BOOL.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal enum BOOL : int
+    {
+        FALSE = 0,
+        TRUE = 1,
```

Comment:
It's possible that other non-zero values could be used to represent true, right?  It doesn't look like it's a problem for this PR, as I don't see any code reading/comparing one of these values marshaled from native, but I'm wondering if having TRUE=1 like this might lead to potential bugs in the future.